container/archive.go

```diff
@@ -13,6 +14,12 @@ import (
 // the absolute path to the resource relative to the container's rootfs, and
 // an error if the path points to outside the container's rootfs.
 func (container *Container) ResolvePath(path string) (resolvedPath, absPath string, err error) {
+	// Check if a drive letter supplied, it must be the system drive. No-op except on Windows
+	err = system.CheckSystemDriveAndRemoveDriveLetter(&path)
```

Comment:
It is called from 2 packages currently.  Hence I put it in a common place. Yes it is specific, but so is what it's doing :)